skyportal/models.py

```diff
@@ -114,7 +122,7 @@ class Source(Base):
 
     score = sa.Column(sa.Float, nullable=True)
 
-    ## pan-starrs
```

Comment:
One of my emacs minor modes is auto-fixing certain formatting issues, haven't been able to pin it down to disable it